### PR TITLE
Adding Tembo logs for docker

### PR DESCRIPTION
**Pull request by joshuajerin**

### CODE REVIEW

File: tembo-cli/src/main.rs [hunk 1]
```diff
@@ -61,6 +63,9 @@ fn main() -> Result<(), anyhow::Error> {
         SubCommands::Delete(_delete_cmd) => {
             delete::execute()?;
         }
+        SubCommands::Logs(logs_cmd) => {
```
Comment (DarrenBaldwin07) [inline tembo-cli/src/main.rs:66]: Likely can remove or set as unused for `logs_cmd`

File: tembo-cli/src/cmd/logs.rs [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+pub fn execute() -> Result<()> {
+    let instance_settings = super::apply::get_instance_settings(None)?;
+    let mut errors = Vec::new();
+    let env = get_current_context()?;
+
+    if env.target == Target::Docker.to_string() {
+        for (instance_name, _settings) in instance_settings {
+            match docker_logs(&instance_name) {
+                Ok(_) => (),
+                Err(e) => errors.push((instance_name, e)),
```
Comment (DarrenBaldwin07) [inline tembo-cli/src/cmd/logs.rs:21]: What are we doing with this `errors` vec? Seems liek we should log the errors at some point

Comment (joshuajerin) [reply tembo-cli/src/cmd/logs.rs:21]: we are printing out the logs too in docker_logs fn
```
let logs_stdout = String::from_utf8_lossy(&output.stdout);
    let logs_stderr = String::from_utf8_lossy(&output.stderr);
```

File: tembo-cli/src/main.rs (hunk 1 above)
Comment (joshuajerin) [reply tembo-cli/src/main.rs:66]: Removed it!

**Pull request closed without merging**